state/internal_test.go

```diff
@@ -76,3 +86,25 @@ func (s *internalStateSuite) TearDownTest(c *gc.C) {
 	s.BaseSuite.TearDownTest(c)
 	s.MgoSuite.TearDownTest(c)
 }
+
+type internalStatePolicy struct{}
```

Comment:
You didn't want to embed Policy in the struct and let any calls to the methods panic instead of spelling out all the methods?